HackIllinois/Assets.xcassets/IndoorMapSiebel0.imageset/Contents.json

```diff
@@ -2,7 +2,7 @@
   "images" : [
     {
       "idiom" : "universal",
-      "filename" : "IndoorMapSiebel3.png",
+      "filename" : "IndoorMapSiebel1.png",
```

Comment:
Can you make it so the asset name matches the file name? In this case it's IndoorMapSiebel0.imageset but the filename is IndoorMapSiebel1.png (the other ones are mismatched as well).